tensorboard/plugins/hparams/constants.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Constants used in the HParams plugin."""
```

Comment:
We've called a similar file metadata.py in most other plugins, see e.g.

https://github.com/tensorflow/tensorboard/blob/1.6.0/tensorboard/plugins/scalar/metadata.py
https://github.com/tensorflow/tensorboard/blob/1.6.0/tensorboard/plugins/custom_scalar/metadata.py